apollo-federation/src/schema/type_and_directive_specification.rs

```diff
@@ -778,6 +787,9 @@ mod tests {
     use crate::schema::type_and_directive_specification::DirectiveSpecification;
     use crate::schema::FederationSchema;
 
+    #[test]
+    fn dead_code_filter() {}
```

Comment:
What does this do?

Reply:
Ah, missed this during my clean up. Rather than mark that module with `allow(dead_code)`, I was going to construct each of the types in this dummy test.